src/modules/dashboard/components/ActionsPage/ActionsComponents/DefaultAction.css

```diff
@@ -50,6 +49,11 @@
 .upperContainer {
   composes: container;
   align-items: baseline;
+  padding-top: 90px;
+}
+
+.bannerPadding {
```

Comment:
I was thinking that bannerPadding may be better added in DefaultMotion. Although, I suppose this allows for other extensions to use it in future.